src/filesystem/emscripten/SDL_sysfilesystem.c

```diff
@@ -85,4 +85,31 @@ SDL_GetPrefPath(const char *org, const char *app)
     return retval;
 }
 
+char *
+SDL_GetPath(SDL_Folder folder)
+{
+    const char *home = NULL;
+    char *retval;
+
+    if (folder != SDL_FOLDER_HOME)
+    {
+        SDL_SetError("Emscripten only supports the home folder");
+        return NULL;
+    }
+
+    home = SDL_getenv("HOME");
+    if (!home)
+    {
+        SDL_SetError("No $HOME environment variable available");
+        return NULL;
+    }
+
+    retval = SDL_strdup(home);
+
+    if (!retval)
+    SDL_OutOfMemory();
```

Comment:
SDL style uses spaces instead of tabs, it looks like this isn't correctly spaced?

Also, SDL style always uses braces, even for single line statements. That makes it easier to add logging or additional logic anywhere that it's needed.